modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="disabling-readiness-init-containers-for-olm-installer_{context}"]
+= Disabling the readiness init containers for the OLM installer
+
+To disable the previously enabled readiness init containers for {prod-short} installed by the Operator Lifecycle Manager (OLM) installer:
+
+.Prerequisites
+
+* {prod-short} is installed by the OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Find the CSV name in the {prod-short} Operator subscription:
```

Comment:
Define CSV here rather than in line 28

Reply:
@themr0c Thank you for catching this, Fabrice.